controllers/socket/lobbyHandlers.go

```diff
@@ -393,3 +393,51 @@ func requestLobbyListDataHandler(_ socketio.Socket) func(string) string {
 		return string(resp)
 	}
 }
+
+var debugLobbyFillFilter = chelpers.FilterParams{
+	FilterLogin: true,
+	Params: map[string]chelpers.Param{
+		"id": chelpers.Param{Kind: reflect.Uint},
+	},
+}
+
+func debugLobbyFillHandler(so socketio.Socket) func(string) string {
+	return chelpers.FilterRequest(so, debugLobbyFillFilter,
+		func(param map[string]interface{}) string {
+			id := param["id"].(uint)
+			lobby, _ := models.GetLobbyById(id)
+			var players []*models.Player
+			userSteamid := chelpers.GetSteamId(so.Id())
+			p, _ := models.GetPlayerBySteamId(userSteamid)
+			players = append(players, p)
+
+			for i := 1; i < models.TypePlayerCount[lobby.Type]*2; i++ {
+				randBytes := make([]byte, 6)
+				rand.Read(randBytes)
+				steamid := "DEBUG" + base64.URLEncoding.EncodeToString(randBytes)
```

Comment:
This would crash if UseSteamAPI were on (because it would fail to extract info from that steam id), but I guess it's fine for debugging